Modelica/Constants.mo

```diff
@@ -32,6 +32,7 @@ package Constants
   // The value for mu_0 can now be expressed as 2*alpha*h/(q^2*c),
   // where alpha is the experimental fine-structure constant,
   // and the value is from https://physics.nist.gov/cuu/pdf/wall_2018.pdf
+  // Note that mu_0 used to be exactly 4*pi*1e-7 H/m but is now 4*pi*1.00000000055(15)*1e-7 H/m
```

Comment:
When I look at the diff of this PR, I don't understand this comment.  The diff suggests that 4*pi*1.00000000055(15)*1e-7 was the previous value.

Reply:
```suggestion
  // A historical definition of mu_0 was that it was equal to exactly 4*pi*1e-7 H/m, but the current definition is 1.00000000055(15) times bigger.
```

Reply:
I will reformulate.
Between 1949 and 2019 it was defined to be 4*pi*1e-7 H/m, before that it was a mess with varying definitions, "historical" seems odd for that period. (Note that the SI-system is from 1960 - but work started in 1948).
Additionally it is now based on experiments, and not defined.

However, that isn't super-important as the main change is that since pi now has a unit we need some way of avoiding a unit-incompatibility and that is easiest done by just giving a value - so this PR doesn't change the value, only the unit.